common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -210,22 +211,30 @@ def test_enroll(self, course_modes, enrollment_mode):
         self.assertTrue(is_active)
         self.assertEqual(course_mode, enrollment_mode)
 
-    def test_enroll_by_email(self):
+    def test_enroll_by_email(self, invitation_only=False):
         """
         Test enroll through API by email.
         """
+        if invitation_only is True:
```

Comment:
`if invitation_only:` might suffice I guess

Reply:
In my opinion it's better to explicitly check boolean, particularly in this function invitation_only may be any type